app/views/facility_orders/show.html.haml

```diff
@@ -1,6 +1,9 @@
 - content_for :h1 do
   = current_facility
 
+= content_for :head_content do
+  = javascript_include_tag "order_table_refresh"
```

Comment:
I'd vote for including this javascript in the application.js package, and not having to download a special JS file for this view...instead, have that javascript only enable itself if a certain element-type is present on the page.

I think it already does this...since if `#results-file-upload-modal` isn't present, nothing will be bound, right?